activestorage/app/jobs/active_storage/transform_job.rb

```diff
@@ -7,6 +7,6 @@ class ActiveStorage::TransformJob < ActiveStorage::BaseJob
   retry_on ActiveStorage::IntegrityError, attempts: 10, wait: :polynomially_longer
 
   def perform(blob, transformations)
-    blob.representation(transformations).processed
+    blob.representation(transformations).processed.key
```

Comment:
Since `representation` makes the assumption that it returns an object that responds to `processed`, I figured it wouldn't hurt to add this `key` call
@jonathanhefner

Reply:
(also key and url are publicly documented methods are usually used after calling `processed` cf. guide )